bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/page/component.tsx

```diff
@@ -231,7 +231,7 @@ const ChatListPage: React.FC<ChatListPageProps> = ({
         const previousMessage = messagesArray[index - 1];
         return (
           <ChatMessage
-            key={message.createdAt}
+            key={message.messageSequence}
```

Comment:
IMHO, you should either add a prefix (e.g., chat-message-${message.messageSequence}) or use the messageId. The key serves as an identifier in React to differentiate elements in the virtual DOM. Using only a number could lead to issues if a new list is added, as it may cause key conflicts.